lib/features/children/generosity_challenge/widgets/picture_attachment_buttons.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:givt_app/core/enums/enums.dart';
+import 'package:givt_app/features/children/generosity_challenge/cubit/generosity_challenge_cubit.dart';
+import 'package:givt_app/features/family/shared/widgets/buttons/givt_elevated_secondary_button.dart';
+import 'package:givt_app/utils/utils.dart';
+
+class PictureAttachmentButtons extends StatelessWidget {
+  const PictureAttachmentButtons({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final cubit = context.read<GenerosityChallengeCubit>();
+    return Column(
+      children: [
+        // WIP: Bug: pops context for some reason when using camera
```

Comment:
Not really for this PR but do you really see a "pop" happening if you for example add a Popscope and debug the method? I do also remember something with "low-memory" and the app process getting killed and restarted when opening and closing the camera.